agent/main.go

```diff
@@ -40,7 +41,12 @@ var extensions []*tapApi.Extension             // global
 var extensionsMap map[string]*tapApi.Extension // global
 
 func main() {
-	logger.InitLoggerStderrOnly()
+	logLevel := logging.INFO
```

Comment:
maybe extract to func

Reply:
i don't think that the functions that return the logging level are necessary

Reply:
@RoyUP9 done with https://github.com/up9inc/mizu/commit/f502187f81c2533f29726eaa0860493521842670 Please review it once more.